cpp/src/arrow/stl_test.cc

```diff
@@ -245,6 +245,26 @@ TEST(TestTableFromTupleVector, ListType) {
   ASSERT_TRUE(expected_table->Equals(*table));
 }
 
+TEST(TestTableFromTupleVector, FixedSizeListType) {
```

Comment:
This adds a test for `TableFromTupleRange`, but can you also add a test for `TupleRangeFromTable`?

Reply:
I will add this test in a moment.